pkg/volume/cinder/attacher.go

```diff
@@ -72,6 +72,14 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 	if err != nil {
 		return "", err
 	}
+	HasDone, VolumeStatus, err := attacher.cinderProvider.OperateHasDone(volumeID)
```

Comment:
Fix the naming of these variables please. Make sure they start with lower case letters.